modern/src/utils.js

```diff
@@ -9,6 +9,14 @@ export function isPromise(obj) {
   return obj && typeof obj.then === "function";
 }
 
+export function isString(obj) {
+  return obj && typeof obj === "string";
```

Comment:
whoops! Moved the condition into the function because I was adding it with every usage, but guess I should have thought that one through more hahah, thanks!